tutorials/common/CMakeLists.txt

```diff
@@ -0,0 +1,12 @@
+file( GLOB_RECURSE sources "${CMAKE_CURRENT_SOURCE_DIR}/*.c" "${CMAKE_CURRENT_SOURCE_DIR}/*.cpp" )
```

Comment:
don't use wildcard, please. You don't have too many files, just list them explicitly:
set(sources
    src/cmd_options.cpp
    src/common_utilities.cpp)